### PR TITLE
Check value is a number rather than a truthy value so that 0 is accepted

### PR DESCRIPTION
**Pull request by lauriejones**
Addresses #17 

I am just checking that `value` is a number as the min/max strings are taken care of by other cases in the switch statement.

I cannot figure out how to update the snapshot. 😅 
`yarn test -u`
`yarn test -- -u`
`yarn test -- --updateSnapshot`

All just throw 
```sh
Unknown argument: updateSnapshot
```

### CODE REVIEW

**Comment by jameslnewell** [comment]
You can't use the `visible` prop for that?

File: src/mixins/gridUnit.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function size({ size }: { size: BreakpointValues<Size> }) {
         `;
 
       default: {
-        const pct = Math.round((value || 1) * 100 * 10000) / 10000; //round to 4 decimal places
+        const pct = Math.round((typeof value === 'number' ? value : 1) * 100 * 10000) / 10000; //round to 4 decimal places
```
[the remark this answers is not in the document]
Comment (jameslnewell) [reply src/mixins/gridUnit.js]: that's stoopid!

**Comment by jameslnewell** [comment]
published

**Comment by lauriejones** [comment]
We probably can!

Thanks for publishing! 🚀

**Pull request merged**